kie-server-parent/kie-server-wars/kie-server/src/main/assembly/assembly-ee8-container.xml

```diff
@@ -91,9 +91,13 @@
 
         <include>org.glassfish.jaxb:jaxb-runtime</include>
         <include>org.glassfish.jaxb:jaxb-xjc</include>
+
+        <include>org.jboss.resteasy:resteasy-jackson2-provider</include>
```

Comment:
Adding resteasy-jackson2-provider to be able to create a custom Rest endpoint with json conversion.